flask_app/templates/project.html

```diff
@@ -14,63 +14,105 @@
 <body>
     <nav class="d-flex justify-content-end mt-3 me-5">
         <ul class="text-end d-inline-flex">
-            <li class="mx-2"><a href="/dashboard">Home </a></li> |
-            <li class="mx-2"><a href="/post">Post</a></li> |
+            <li class="mx-2"><a href="/post">Home </a></li> |
+            <li class="mx-2"><a href="/project">Post</a></li> |
             <li class="mx-2"><a href="/logout">Logout</a></li> 
         </ul>
     </nav>
-    <main class=" mx-auto mt-5 col-6 d-flex flex-column align-items-center">
-        <h1 class="text-center">{Project Name} Tasks</h1>
-        <!-- Should we display anything else about the task? -->
-        <div class="my-3">
-            <a class="btn btn-success" href="/status/{{this_proeject_id}}">Check Project Status</a>
+    <main class=" mx-auto mt-5 col-6 d-flex flex-column">
+        <!-- Project info, un-task related -->
+        <div>
+            <!-- <h1 class="">{Project Name} Tasks</h1>
+            {% if project.category == 'Literature'  %}
+                <h2> Subject: Literature <i class="bi bi-journal-text"></i></h2>
+            {% elif %}
+                <h2> Subject: History <i class="bi bi-globe-europe-africa"></i></h2>
+            {% elif %}
+                <h2> Subject: Social Science<i class="bi bi-bar-chart-line-fill"></i></h2>
+            {% elif %}
+                <h2> Subject: Science <i class="bi bi-lightning-charge-fill"></i></h2>
+            {% elif %}
+                <h2> Subject: Art <i class="bi bi-brush"></i></h2>
+            {% end if %}
+            {% if  {{ time.now.date() }} >= project.due_date - time.delta(days=2) %}
```

Comment:
Pretty cool idea here!

Reply:
I will need to be able to pull from the database and work within the virtual environment to test. Hoping we can pull it off.